sql-cli/sql_cli/project.py

```diff
@@ -76,22 +82,34 @@ def airflow_config(self) -> dict[str, Any]:
         parser.read(filename)
         return {section: dict(parser.items(section)) for section in parser.sections()}
 
-    def update_config(self, environment: str = DEFAULT_ENVIRONMENT) -> None:
+    def _initialise_global_config(self) -> None:
         """
-        Sets custom Airflow configuration in case the user is not using the default values.
+        Initialises global config file that includes configuration to be shared across environments including the
+        airflow config.
+        """
+        config = Config(environment=GLOBAL_CONFIG, project_dir=self.directory)
+        global_env_filepath = config.get_global_config_filepath()
+        config.write_value_to_yaml("airflow", "home", str(self._airflow_home.resolve()), global_env_filepath)
```

Comment:
Should we also validate here that the airflow home exists?

Reply:
Right, thank you :) But can we add that as a comment please :)

Reply:
Added.